### PR TITLE
Fix wake induced velocities

### PR DESCRIPTION
**Pull request by rachealerhard**
# Pull Request Checklist
 * Merge in the latest Develop branch changes to your branch
 * Remove .pyc files from your repository:
   * Linux/Mac
     * find . -name *.pyc -delete
   * Windows
     * del /S *.pyc
 * Run automatic regression and makes sure everything is passing
 * Did you make a new regression test that covers your new code?
 * Did you update your docstrings?
 * Did you update your headers to include your name and date?
 * Do a final compare with the Develop branch to be sure what you're changing

### CODE REVIEW

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/extract_wing_VD.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    VD   = geometry.vortex_distribution
+    n_sw = VD.n_sw 
+    n_cw = VD.n_cw  
+
+    VD_wing = Data()    
+    vd_i    = 0        # count of current VD wing elements
+    j       = 0        # count of current VD wing index
+    size = n_cw * n_sw
+    
+    for idx,wing in enumerate(geometry.wings):
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/extract_wing_VD.py:44]: I'm confused why you even need this whole function? Maybe we can talk about indexing through the VLM. Aaron and I have a nice way of splitting out wings if that's what this is trying to do.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/extract_wing_VD.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    Outputs:
+    None
+
+    Properties Used:
+    N/A
+    """
+    # unpack vortex distribution properties
+    VD   = geometry.vortex_distribution
+    n_sw = VD.n_sw
```
[the remark this answers is not in the document]
Comment (rachealerhard) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/extract_wing_VD.py:36]: this gives the array of n_sw for each wing (or half wing for symmetric wings) so it should work as intended here

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/extract_wing_VD.py (hunk 1 above)
Comment (planes) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/extract_wing_VD.py:44]: The VLM includes the indices of where a wing stops and the next begins. [You can see an example here where it splits](https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/VLM.py#L480). Just remember the VLM works on semispan wings.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/extract_wing_VD.py [hunk 3]
```diff
@@ -9,58 +9,45 @@
...
     # unpack vortex distribution properties
-    VD   = geometry.vortex_distribution
-    n_sw = VD.n_sw 
-    n_cw = VD.n_cw  
-
-    VD_wing = Data()    
-    vd_i    = 0        # count of current VD wing elements
-    j       = 0        # count of current VD wing index
-    size = n_cw * n_sw
+    VD          = geometry.vortex_distribution
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/extract_wing_VD.py:37]: So much cleaner

File: trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/import_airfoil_polars.py [hunk 4]
```diff
@@ -39,8 +39,8 @@ def  import_airfoil_polars(airfoil_polar_files):
     num_polars   = 0
     for i in range(num_airfoils): 
         n_p = len(airfoil_polar_files[i])
-        if n_p < 3:
-            raise AttributeError('Provide three or more airfoil polars to compute surrogate')
+        #if n_p < 3:
```
Comment (planes) [inline trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/import_airfoil_polars.py:42]: commented out code

File: trunk/SUAVE/Plots/Performance/Airfoil_Plots.py [hunk 5]
```diff
@@ -308,6 +308,99 @@ def plot_airfoil_polar_files(airfoil_path, airfoil_polar_paths, line_color = 'k-
...
+
+    col_raw = ['m-', 'b-', 'r-', 'g-', 'o-','p-']    
+    if use_surrogate:
+        col_sur = ['m--', 'b--', 'r--', 'g--', 'o--','p--']
+        # Compute airfoil surrogates
+        a_data = compute_airfoil_polars(airfoil_path, airfoil_polar_paths,npoints = 200, use_pre_stall_data=False)
+        CL_sur = a_data.lift_coefficient_surrogates
+        CD_sur = a_data.drag_coefficient_surrogates
+        
+        alpha   = np.linspace(-16,16,100) #np.asarray(a_data.aoa_from_polar)
```
Comment (planes) [inline trunk/SUAVE/Plots/Performance/Airfoil_Plots.py:349]: commented code

**Review by planes — COMMENTED** [review]
So close

File: trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/import_airfoil_polars.py (hunk 4 above)
Comment (rachealerhard) [reply trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/import_airfoil_polars.py:42]: oops, thanks!

File: trunk/SUAVE/Plots/Performance/Airfoil_Plots.py (hunk 5 above)
Comment (rachealerhard) [reply trunk/SUAVE/Plots/Performance/Airfoil_Plots.py:349]: fixed

**Pull request merged**